UnitTestFrameworkPkg/GoogleTest.md

```diff
@@ -687,6 +726,8 @@ declarations as shown in the below table.
 | Global Table (e.g. gRT, gBS, etc.) | gRT->GetVariable | gRT_GetVariable |
 | Protocol | UsbIoProtocol->UsbPortReset | UsbIoProtocol_UsbPortReset |
 
+__Note__ the naming convention for global tables and protocols is currently being discussed. Points of discussion include removing the use of underscores, use of "Efi," and including the use case, e.g. `Protocol` in the mock function name.

```

Comment:
@mdkinney Here is a note on the naming convention

Reply:
@mdkinney This fell off my radar, but I wanted to get your thoughts on this - can you give any feedback in this PR, please?

Reply:
I provided feedback in the tools meeting on additional topics that needed to be covered.  I have not seen any updates.  Did I miss them?

Reply:
To close on this. What are the remaining opens and the status?

Reply:
My list was
- Adding Instructions/example for creating mocks of non-libraries (Protocols, PPIs, etc.)
- Updating file location/file name rules
- Adding a section on mocking variadic functions

Which are all included in this PR

Reply:
@mdkinney, can you please check if this thread can be closed now?

Reply:
After the content below is accepted, the `note` above should be removed.